commerce-add-ons/payment-providers/README.md

```diff
@@ -1,19 +1,19 @@
 ---
 description: >-
-  Learn more about each of the available integrations for implementing payment
-  providers into your Umbraco Commerce installation.
+  Learn about the available integrations for implementing payment
+  providers in your Umbraco Commerce installation.
 ---
 
 # What are Payment Providers?
 
-Payment providers are essential for your Umbraco Commerce installation when you want to give users and customers to option to pay for your products.
+Payment providers are essential for your Umbraco Commerce installation when you want to give users and customers the option to pay for your products.
 
-The [Invoice](invoicing/) and [Zero](zero/) Payment Providers comes with Umbraco Commerce out of the box. Nine pre-built integrations for different providers are available for you to add additional payment options to your site.
+By default, Umbraco Commerce includes the [Invoice](invoicing/) and [Zero](zero/) payment providers. Additionally, nine pre-built integrations are available to expand your payment options.
```

Comment:
Instead of having to keep track of this number whenever we add new payment providers, how about we change the wording a little? 🤔 

```suggestion
By default, Umbraco Commerce includes the [Invoice](invoicing/) and [Zero](zero/) payment providers. Additionally, more pre-built integrations are available to expand your payment options.
```